### PR TITLE
perf(specParser): error handling, localization

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

**Review by nliu-ms — DISMISSED** [review]

**Review by adashen — APPROVED** [review]

**Review by yuqizhou77 — APPROVED** [review]

**Review by Alive-Fish — DISMISSED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -374,6 +374,16 @@
   "core.common.OutlookDesktopClientName": "Outlook desktop client id",
   "core.common.OutlookWebClientName1": "Outlook web access client id 1",
   "core.common.OutlookWebClientName2": "Outlook web access client id 2",
+  "core.common.CancelledMessage": "Operation cancelled.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:377]: cancelled -> is cancelled.

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -374,6 +374,16 @@
...
+  "core.common.SpecVersionNotSupported": "OpenAPI version lower than 3.0.0 is not supported.",
+  "core.common.NoServerInformation":  "No server information found in the OpenAPI specification file.",
+  "core.common.MultipleServerInformation": "Multiple server information found in the OpenAPI specification file.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:380]: found -> is found

File: packages/fx-core/resource/package.nls.json [hunk 3]
```diff
@@ -374,6 +374,16 @@
...
+  "core.common.RemoteRefNotSupported": "Remote reference is not supported: %s.",
+  "core.common.MissingOperationId": "Missing operationIds: %s.",
+  "core.common.NoSupportedApi": "No supported API found in the OpenAPI specification file.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:383]: found -> is found

File: packages/fx-core/resource/package.nls.json [hunk 4]
```diff
@@ -374,6 +374,16 @@
...
+  "core.common.AdditionalPropertiesNotSupported": "additionalProperties is not supported, and will be ignored.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:384]: this additionalProperties is a dedicated term, in order to use the singular form, better to quote it like "additionalProperties" or 'additionalProperties'

File: packages/fx-core/resource/package.nls.json [hunk 5]
```diff
@@ -374,6 +374,16 @@
...
+  "core.common.SchemaNotSupported": "oneOf, anyOf, and not schema is not supported: %s.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:385]: not schema is a term? then they need to be quoted, and the same applies to oneOf and anyOf.

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (SLdragon) [reply packages/fx-core/resource/package.nls.json:377]: Thanks, will update

**Review by yuqizhou77 — DISMISSED** [review]

**Review by Alive-Fish — APPROVED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 6]
```diff
@@ -380,6 +380,16 @@
...
+  "core.common.NoServerInformation":  "No server information is found in the OpenAPI specification file.",
+  "core.common.MultipleServerInformation": "Multiple server information is found in the OpenAPI specification file.",
+  "core.common.RemoteRefNotSupported": "Remote reference is not supported: %s.",
+  "core.common.MissingOperationId": "Missing operationIds: %s.",
+  "core.common.NoSupportedApi": "No supported API is found in the OpenAPI specification file.",
+  "core.common.AdditionalPropertiesNotSupported": "'additionalProperties' is not supported, and will be ignored.",
+  "core.common.SchemaNotSupported": "'oneOf', 'anyOf', and 'not' schema are not supported: %s.",
+  "core.common.UnknownSchema": "Unknown schema: %s.",
   "core.importAddin.label": "Import an Existing Outlook Add-in",
   "core.importAddin.detail": "Upgrade and add-in project to the latest app manifest and project structure",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:394]: Upgrade and -> Upgrade an?

Alo please check the official doc, how the Outlook Add-In is capitalized. whether it is Outlook Add-In or Add-in or add-in, then update this line and the previous line for it.

Comment (SLdragon) [reply packages/fx-core/resource/package.nls.json:394]: Thanks, from [doc](https://support.microsoft.com/en-us/office/get-an-office-add-in-for-outlook-1ee261f9-49bf-4ba6-b3e2-2ba7bcab64c8) it is: Add-ins

**Review by sffamily — APPROVED** [review]

**Review by yuqizhou77 — APPROVED** [review]

**Review by adashen — APPROVED** [review]

**Pull request merged**